README.md

```diff
@@ -1,55 +1,70 @@
 # cfgov-refresh
 
-This repository contains the redesign-in-progress of consumerfinance.gov. This includes front-end assets and build tools, and configuration for [Sheer](https://github.com/cfpb/sheer) to load content from Wordpress and Django back-ends to elasticsearch to render the site.
+**Description**: This repository contains the redesign-in-progress of the [consumerfinance.gov](http://consumerfinance.gov) website. This includes front-end assets and build tools, and configuration for [Sheer](https://github.com/cfpb/sheer) to load content from WordPress and Django back-ends to Elasticsearch to render the site.
 
-### This project is a work in progress
+**Technology stack**:
```

Comment:
Point taken, but it's pretty trivial to have the instructions support both OS X and other Unix platforms, whereas explaining how to get it up and running on a Windows machine is an order of magnitude harder. I feel like most open source projects that are developed on Macs tend to write their instructions to generically apply to Unix-based systems, but maybe I'm wrong.

I think ultimately, I'm rehashing what you've already opened up in cfpb/open-source-project-template#11. We should get that sorted out.